filebeat/input/filestream/input.go

```diff
@@ -43,6 +45,8 @@ const pluginName = "filestream"
 
 type state struct {
 	Offset int64 `json:"offset" struct:"offset"`
+
+	id string
```

Comment:
The state is going to be stored in the regsitry file. Do we really need the ID here? Does the ID change?